mne/viz/evoked.py

```diff
@@ -1082,7 +1111,7 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     ts_args_def = dict(picks=None, unit=True, ylim=None, xlim='tight',
                        proj=False, hline=None, units=None, scalings=None,
                        titles=None, gfp=False, window_title=None,
-                       spatial_colors=True)
+                       spatial_colors=True, zorder='std')
```

Comment:
What about this? I'm fine with changing the default to 'std', but now the docs are kind of contradicting with this.